app/server/api/static/apply-email-confirmation.yml

```diff
@@ -0,0 +1,7 @@
+---
+contentType: page
+useComponent: true
+slug: apply-email-confirmation
+image: "/ui/img/content/registration-handshake-email.png"
```

Comment:
Do these need to be uploaded yet?

Reply:
They are in the /ui/img/content/ folder so _should_ go up automatically. Are they missing from your build? @Koded